File: app/imports/api/person/person.js
import { Mongo } from 'meteor/mongo';
import SimpleSchema from 'simpl-schema';

const PersonSchema = new SimpleSchema({
	firstName: {
		type: String,
		optional: true
	},
	middleName: {
		type: String,
		optional: true
	},
	lastName: {
		type: String,
		optional: true
	},

	nickNames: {
		type: Array,
		optional: true
	},
	"nickNames.$": {
		type: String
	},

	phoneNumbers: {
		type: Array,
		optional: true
	},
	"phoneNumbers.$": {
		type: String
	},

	emailAddresses: {
		type: Array,
		optional: true
	},
	"emailAddresses.$": {
		type: String
	},

	organizations: {
		type: Array,
		optional: true
	},
	"organizations.$": {
		type: String
	},

	employers: {
		type: Array,
		optional: true
	},
	"employers.$": {
		type: String
	},

	socialLinks: {
		type: Array,
		optional: true
	},
	"socialLinks.$": {
		type: Object
	},
	"socialLinks.$.site": {
		type: String
	},
	"socialLinks.$.url": {
		type: String
	},

	/* Relationships */
	relations: {
		type: Array,
		optional: true
	},
	"relations.$": {
		type: Object
	},
	"relations.$.collectionName": {
		type: String,
		optional: true
	},
	"relations.$.collectionId": {
		type: String,
		optional: true
	}
});

const Person = new Mongo.Collection('person');
Person.attachSchema(PersonSchema);

export default Person;

File: poc/SignalRTest/SignalRTest/DataAccess/Models/Person.cs
using NPoco;
using SignalRTest.DataAccess.Attributes;
using SignalRTest.DataAccess.Models.Attributes;
using System;
using System.Collections.Generic;

namespace SignalRTest.DataAccess.Models
{
    [TableName("people")]
    [PrimaryKey("id")]
    public class Person
    {
        [Column("id")] public Guid Id { get; set; }

        [Column("firstname")] public string FirstName { get; set; }
        [Column("middlename")] public string MiddleName { get; set; }
        [Column("lastname")] public string LastName { get; set; }
        [Column("nicknames")] public IEnumerable<string> Nicknames { get; set; }
        [Column("phonenumbers")] public IEnumerable<PhoneNumber> PhoneNumbers { get; set; }
        [Column("emailaddresses")] public IEnumerable<string> EmailAddresses { get; set; }
        [Column("organizations")] public IEnumerable<string> Organizations { get; set; }
        [Column("employers")] public IEnumerable<string> Employers { get; set; }
        [Column("sociallinks")] public IEnumerable<SocialLink> SoclalLinks { get; set; }

        [Column("relations")] public IEnumerable<Relation> Relations { get; set; }
        [Column("datecreated")] public DateTime DateCreated { get; set; }
        [Column("lastupdated")] public DateTime LastUpdated { get; set; }
    }
}
